scripts/SANS/sans/gui_logic/presenter/run_tab_presenter.py

```diff
@@ -878,6 +882,17 @@ def on_save_other(self):
         self.save_other_presenter.set_view(save_other_view)
         self.save_other_presenter.show()
 
+    def on_reduction_mode_selection_has_changed(self, selection):
+        if not selection:
+            return
+        mode = get_reduction_mode_from_gui_selection(selection)
+        if mode == ISISReductionMode.HAB:
+            self._beam_centre_presenter.update_hab_selected()
+        elif mode == ISISReductionMode.LAB:
+            self._beam_centre_presenter.update_lab_selected()
+        else:
+            self._beam_centre_presenter.update_all_selected()
+
     # def _validate_rows(self):
     #     """
```

Comment:
If this commented out method is unused, should it be deleted?

Reply:
Git history shows it was commented out at least 11 months ago, so it's probably safe to remove